apps/web/src/pages/PrivacyPolicy/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,206 @@
+    }
+  >
+    {chunks}
+  </Link>
+);
+
+const PrivacyPolicy = () => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+
+  const pageTitle = intl.formatMessage({
+    defaultMessage: "Privacy policy",
+    id: "cYNDhP",
+    description: "Title for the websites privacy policy",
+  });
+
+  const subtitle = intl.formatMessage({
+    defaultMessage:
+      "Learn more about how GC Digital Talent handles user privacy.",
+    id: "Vz719J",
+    description: "Sub title for the websites privacy policy",
+  });
+
+  const crumbs = useBreadcrumbs([
+    {
+      label: pageTitle,
+      url: paths.privacyPolicy(),
+    },
+  ]);
+
+  return (
+    <>
+      <Hero title={pageTitle} subtitle={subtitle} crumbs={crumbs} />
+      <div
+        data-h2-container="base(center, large, x1)"
+        data-h2-margin="base(x3)"
+      >
+        <Heading size="h3" data-h2-margin="base(0, 0, x1, 0)">
+          {pageTitle}
+        </Heading>
+        <p data-h2-margin="base(x1 0)">
+          {intl.formatMessage(
+            {
+              defaultMessage:
+                "Personal information collected through GC Digital Talent is used for staffing, external recruitment, and internal talent mobility within federal institutions pursuant to <justiceLaws7Link>section 7(1)</justiceLaws7Link> of the Financial Administration Act, <justiceLaws15Link>section 15(1)</justiceLaws15Link>, <justiceLaws29Link>section 29</justiceLaws29Link> and <justiceLaws30Link>30 (1), (2), and (3)</justiceLaws30Link> of the Public Service Employment Act and section 5 of the Employment Equity Act.",
+              id: "e1JW62",
+              description: "Paragraph for privacy policy page",
+            },
+            {
+              justiceLaws7Link: (chunks: React.ReactNode) =>
+                justiceLaws7Link(locale, chunks),
+              justiceLaws15Link: (chunks: React.ReactNode) =>
+                justiceLaws15Link(locale, chunks),
+              justiceLaws29Link: (chunks: React.ReactNode) =>
+                justiceLaws29Link(locale, chunks),
+              justiceLaws30Link: (chunks: React.ReactNode) =>
+                justiceLaws30Link(locale, chunks),
+            },
+          )}
+        </p>
+        <p data-h2-margin="base(x1 0)">
+          {intl.formatMessage({
+            defaultMessage:
+              "Please do not provide additional personal information which is not required for this purpose.",
+            id: "zMa59V",
+            description: "Paragraph for privacy policy page",
+          })}
+        </p>
+        <p data-h2-margin="base(x1 0)">
+          {intl.formatMessage(
+            {
+              defaultMessage:
+                "  The information you provide may also be used for statistical and research purposes, and may be disclosed to the <publicServiceLink>Public Service Commission Investigation Directorate</publicServiceLink> when necessary.",
```

Comment:
This will change the id and the corresponding fr.json file will need to be changed accordingly.
```suggestion
                "The information you provide may also be used for statistical and research purposes, and may be disclosed to the <publicServiceLink>Public Service Commission Investigation Directorate</publicServiceLink> when necessary.",
```